imports/ui/pages/professor/manage_session.jsx

```diff
@@ -221,18 +221,12 @@ class _ManageSession extends Component {
     tags.push({ value: code, label: code })
     tags.push({ value: semester, label: semester })
 
-    const blankQuestion = {
-      plainText: '', // plain text version of question
-      type: -1,
-      content: '', // wysiwyg display content
-      options: [],
-      tags: tags,
+    const blankQuestion = _.extend({   
       sessionId: sessionId,
       courseId: this.state.session.courseId,
       owner: course.owner,
```

Comment:
This is wrong, I think owner should be the current user (e.g. anyone in course.instructors should be able to crate questions and own them).